src/commands/install.yml

```diff
@@ -0,0 +1,53 @@
+description: Install the Rainforest QA CLI
+parameters:
+  install-path:
+    description: Where to install the CLI
+    type: string
+    default: /usr/local/bin
+  channel:
+    description: The version of the CLI to install
+    type: enum
+    enum: ["stable", "beta", "dev"]
+    default: stable
+  platform:
+    description: Which platform to install the CLI for
+    type: enum
+    enum: ["darwin", "linux"]
+    default: linux
+  format:
+    description: |
+      What compression format to download. Linux must use tgz. MacOS (Darwin) can choose either tgz or zip.
+    type: enum
+    enum: ["zip", "tgz"]
+    default: tgz
+  architecture:
+    description: 386 (32-bit) or amd64 (64-bit)?
+    type: enum
+    enum: ["386", "amd64"]
+    default: amd64
+steps:
+  - run:
+      name: Get download URL
+      command: |
+        url="https://bin.equinox.io/c"
+        file="rainforest-cli"
+        case << parameters.channel >> in
+          stable) url="$url/htRtQZagtfg"; file="$file-stable";;
+          beta) url="$url/csTtrGrMBYr"; file="$file-beta";;
+          dev) url="$url/6dz6cwxHnZ8"; file="$file-dev";;
+          *) echo "Invalid channel << parameters.channel >>" 1>&2; exit 1;;
+        esac
+
+        file="$file-<< parameters.platform >>-<< parameters.architecture >>.<< parameters.format >>"
+        url="$url/$file"
+
+        if [ << parameters.format >> = "tgz" ] ; then
+          wget $url
+          tar xvf $file -C << parameters.install-path >>
+        elif [ << parameters.format >> = "zip" && << parameters.platform >> = "darwin" ] ; then
+          curl -O $url
+          unzip $file -d << parameters.install-path >>
```

Comment:
Is there a reason to support zip? If we didn't, then we could delete a bunch of logic here.

Reply:
I originally had windows here as well, and windows was only zip, but there were no CLI install instructions on equinox because...windows. I'll remove 👍